service/internal/resource/config.go

```diff
@@ -28,6 +28,11 @@ func New(buildInfo component.BuildInfo, resourceCfg map[string]*string) *resourc
 		telAttrs = append(telAttrs, attribute.String(semconv.AttributeServiceName, buildInfo.Command))
 	}
 
+	if _, ok := resourceCfg[semconv.AttributeServiceNamespace]; !ok {
+		// AttributeServiceNamespace is not specified in the config. Use the default namespace.
```

Comment:
```suggestion
		// AttributeServiceNamespace is not specified in the config. Use the namespace built into the binary.
```
If possible I think we should specify where the default comes from.